src/components/Editor/Preview/Popup.js

```diff
@@ -103,16 +123,18 @@ export class Popup extends Component<Props> {
   }
 
   renderObjectPreview(expression: string, root: Object, extra: Object) {
```

Comment:
also maybe here we can use the node type for root:  https://github.com/devtools-html/debugger.html/blob/master/src/utils/sources-tree/types.js#L14

it will need to be moved to a more general type

Reply:
yes